tests/Integration/Behaviour/Features/Context/ShopFeatureContext.php

```diff
@@ -49,19 +53,138 @@ public function shopWithNameExists(string $reference, string $shopName)
         SharedStorage::getStorage()->set($reference, new Shop($shopId));
     }
 
+    /**
+     * @Given /I add a shop group "(.+)" with name "(.+)"$/
+     *
+     * @param string $reference
+     * @param string $groupName
+     */
+    public function addShopGroup(string $reference, string $groupName): void
+    {
+        $shopGroup = new ShopGroup();
+        $shopGroup->name = $groupName;
+        $shopGroup->active = true;
+        if (!$shopGroup->add()) {
+            throw new RuntimeException(sprintf('Could not create shop group: %s', Db::getInstance()->getMsgError()));
+        }
+
+        SharedStorage::getStorage()->set($reference, $shopGroup);
+    }
```

Comment:
I added exactly the same method in this PR #21593, but with color too.

Reply:
I know. I copy it for easily merge it in 178. ;)

Reply:
ah nice 👍 one problem less